src/services/ui/src/pages/detail/admin-changes/AdminChanges.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components";
+import { opensearch } from "shared-types";
+import { FC, useMemo } from "react";
+import { BLANK_VALUE } from "@/consts";
+import { format } from "date-fns";
+
+export const AC_WithdrawEnabled: FC<opensearch.changelog.Document> = (
+  props
+) => {
+  return (
+    <p>
+      {props.submitterName} has enabled package action to submit formal RAI
+      response
+    </p>
+  );
+};
+
+export const AC_WithdrawDisabled: FC<opensearch.changelog.Document> = (
+  props
+) => {
+  return (
+    <p>
+      {props.submitterName} has disabled package action to submit formal RAI
+      response
+    </p>
+  );
+};
+
+export const AC_Update: FC<opensearch.changelog.Document> = () => {
+  return <p>Coming Soon</p>;
+};
+
+const useAdminChange = (doc: opensearch.changelog.Document) => {
+  return useMemo(() => {
+    switch (doc.actionType) {
+      case "disable-rai-withdraw":
+        return ["Disabled formal RAI response withdraw", AC_WithdrawDisabled];
```

Comment:
I like bundling these, but would a typed response be of any benefit here? I see we're doing an inline type below on line 59.

Reply:
Added a response type 👍